tests/utils.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+from requests import Session, Response
+from scrapy import Spider
+from scrapy.http import HtmlResponse
+
+from models.point_of_interest import PointOfInterest
+
+
```

Comment:
One other option would be to refactor:

```python
def parse_betamax_response(session: Session, parse_function: Callable, url: str) -> List[PointOfInterest]:
    """
    Loads response for a given URL through Betamax  and parses it with a given spider
    """
    scrapy_response = get_betamax_response(session, url)
    results = [PointOfInterest(**result) for result in parse_function(scrapy_response)]

    return results
```

Reply:
Hm… idk, sound a bit like an overcomplicating it…

Reply:
You're right, this does make it more concise!